flow360/component/flow360_params/flow360_output.py

```diff
@@ -0,0 +1,323 @@
+    vorticity: Optional[bool] = pd.Field()
+    wall_distance: Optional[bool] = pd.Field(alias="wallDistance")
+    low_numerical_dissipation_sensor: Optional[bool] = pd.Field(
+        alias="lowNumericalDissipationSensor"
+    )
+    residual_heat_solver: Optional[bool] = pd.Field(alias="residualHeatSolver")
+
+
+OutputFormat = Union[Literal["paraview"], Literal["tecplot"], Literal["both"]]
+
+
+class Surface(Flow360BaseModel):
+    """:class:`Surface` class"""
+
+    output_fields: Optional[List[str]] = pd.Field(alias="outputFields")
+
+
+class _GenericSurfaceWrapper(Flow360BaseModel):
+    """:class:`_GenericSurfaceWrapper` class"""
+
+    v: Surface
+
+
+class Surfaces(Flow360SortableBaseModel):
+    """:class:`Surfaces` class"""
+
+    @classmethod
+    def get_subtypes(cls) -> list:
+        return [_GenericSurfaceWrapper.__fields__["v"].type_]
+
+    # pylint: disable=no-self-argument
+    @pd.root_validator(pre=True)
+    def validate_surface(cls, values):
+        """
+        root validator
+        """
+        return _self_named_property_validator(
+            values, _GenericSurfaceWrapper, msg="is not any of supported surface types."
+        )
+
+
+class SurfaceOutput(OutputBase):
+    """:class:`SurfaceOutput` class"""
+
+    output_format: Optional[OutputFormat] = pd.Field(alias="outputFormat")
+    animation_frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
+        alias="animationFrequency"
+    )
+    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
+    animation_frequency_time_average: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
+        alias="animationFrequencyTimeAverage"
+    )
+    animation_frequency_time_average_offset: Optional[int] = pd.Field(
+        alias="animationFrequencyTimeAverageOffset"
+    )
+    compute_time_averages: Optional[bool] = pd.Field(alias="computeTimeAverages")
+    velocity_relative: Optional[bool] = pd.Field(alias="VelocityRelative")
+    write_single_file: Optional[bool] = pd.Field(alias="writeSingleFile")
+    primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
+    start_average_integration_step: Optional[bool] = pd.Field(alias="startAverageIntegrationStep")
+    output_fields: Optional[List[Union[CommonFieldVars, SurfaceFieldVars]]] = pd.Field(
+        alias="outputFields"
+    )
+    surfaces: Optional[Surfaces] = pd.Field()
+
+    Cf: Optional[bool] = pd.Field(alias="Cf")
+    Cf_vec: Optional[bool] = pd.Field(alias="CfVec")
+    Cf_normal: Optional[bool] = pd.Field(alias="CfNormal")
+    Cf_tangent: Optional[bool] = pd.Field(alias="CfTangent")
+    y_plus: Optional[bool] = pd.Field(alias="yPlus")
+    wall_distance: Optional[bool] = pd.Field(alias="wallDistance")
+    heat_flux: Optional[bool] = pd.Field(alias="heatFlux")
+    node_forces_per_unit_area: Optional[bool] = pd.Field(alias="nodeForcesPerUnitArea")
+    node_normals: Optional[bool] = pd.Field(alias="nodeNormals")
```

Comment:
move all bool options to separate class that will be only used in updater but not be part of allowed configuration

Reply:
Fixed in 05ca5e6